src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Rectangle implements Figure {
```

Comment:
The class 'Rectangle' should not start with an empty line. Please remove the empty line before the class declaration.